### PR TITLE
Refactor the six cows into a list of objects.

### PR DESCRIPTION
**Pull request by tttppp**

### CODE REVIEW

File: MovingSquares/app/src/main/java/l00lgamescommunity/movingsquares/MainActivity.java [hunk 1]
```diff
@@ -1,73 +1,50 @@
...
-import android.content.Context;
 import android.content.Intent;
-import android.media.MediaPlayer;
-import android.os.CountDownTimer;
 import android.os.Bundle;
-import android.view.Menu;
-import android.view.MenuItem;
+import android.os.CountDownTimer;
 import android.view.MotionEvent;
-import android.view.View;
```
Comment (tttppp) [inline MovingSquares/app/src/main/java/l00lgamescommunity/movingsquares/MainActivity.java]: I'm pretty sure some of these imports are *import*ant. Oops.

File: MovingSquares/app/src/main/java/l00lgamescommunity/movingsquares/Cow.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+public class Cow {
+    private float xPos = 0;
+    private float yPos = 0;
+    private float initialX = 0;
+    private boolean dead = false;
+    private int imageId;
+    private int deadImageId;
+    private int score;
+
+    public Cow(int imageId, int deadImageId, int score) {
```
Comment (nyoany) [inline MovingSquares/app/src/main/java/l00lgamescommunity/movingsquares/Cow.java:15]: Cow should not have score in constructor.  That is the speed :dancing_men:

Comment (tttppp) [reply MovingSquares/app/src/main/java/l00lgamescommunity/movingsquares/Cow.java:15]: I think it's the score. It's the number of points for saving the cow. The speed doesn't vary between cows, its currently an attribute of the level difficulty.

**Review by tttppp — COMMENTED** [review]
Didn't realise these comments were "Pending". Submitting them for visibility.

**Pull request merged**